Fix a divide-by-zero-error in UsagePlot.cpp. Will be
fixed in MultiImonC 0.3.3.

--- UsagePlot.cpp.orig	2004-06-02 21:59:34.973014648 +0200
+++ UsagePlot.cpp	2004-06-02 22:03:14.151694424 +0200
@@ -123,18 +123,23 @@
         int iOldYOut = 0;
         ChannelUsageData::const_iterator itData((*cli).begin());
         while(itData != (*cli).end()) {
-            wxCoord x(iElement * iHorizSpace);
-            wxCoord iYIn(((iHeight - 6) * (*itData).in / (*cli).MaxRateIn) + 1);
-            wxCoord iYOut(((iHeight - 6) * (*itData).out / (*cli).MaxRateOut) + 1);
-            
-            dc.SetPen((*cli).PenColorOut());
-            dc.DrawLine(x - iHorizSpace, iOldYOut, x, iYOut);
-            
-            dc.SetPen((*cli).PenColorIn());
-            dc.DrawLine(x - iHorizSpace, iOldYIn, x, iYIn);
-            
-            iOldYIn = iYIn;
-            iOldYOut = iYOut;
+            ChannelUsageData const& oUsage(*cli);
+
+            if(oUsage.MaxRateIn && oUsage.MaxRateOut) {
+                wxCoord x(iElement * iHorizSpace);
+                wxCoord iYIn(((iHeight - 6) * (*itData).in / oUsage.MaxRateIn) + 1);
+                wxCoord iYOut(((iHeight - 6) * (*itData).out / oUsage.MaxRateOut) + 1);
+
+                dc.SetPen(oUsage.PenColorOut());
+                dc.DrawLine(x - iHorizSpace, iOldYOut, x, iYOut);
+
+                dc.SetPen(oUsage.PenColorIn());
+                dc.DrawLine(x - iHorizSpace, iOldYIn, x, iYIn);
+
+                iOldYIn = iYIn;
+                iOldYOut = iYOut;
+            }
+			
             iElement++;
             itData++;
         }
